### ISSUE TITLE
Can't call start() twice

### ISSUE THREAD

**Issue by raybooysen** [KidkArolis/location-bar#5]
The usage of location-bar is slightly odd.  You initialise a new instance:

`var bar = new LocationBar()`

Yet when you call start, this checks the `History.started` field which is global.  This is slighly odd as the ability to instanciate multiple instances of LocationBar gives the illusion that you can create and start multiple instances.

Otherwise, great product.

**Comment by KidkArolis** [comment]
An interesting point. I guess I just kept it in how they did in Backbone without thinking about it too much. In Backbone they actually [instantiate it once and assign it to Backbone.history](https://github.com/jashkenas/backbone/blob/master/backbone.js#L1807-L1808) - so there's always only once instance of this per page.

Hm, so there are two ways this could possibly be improved.

We could convert locationBar into a singleton, but I don't quite like this option, because a LocationBar instance contains state (the routes that get added, or the current fragment info, etc.). So I like the class > instance pattern as opposed to directly doing `require('location-bar').onChange`.

The History.started flag could then just be seen as some kind of lock preventing you from .. creating two instances of LocationBar since that's most likely not what you want. I think in most cases it doesn't make sense to have two locationBar objects, since there's only 1 location bar in the browser.

Having said that, it could be interesting to have different parts of the application have their own location bar instances with different sets of routes. That means they can kind of match different aspects of the URL - one could match some query param, one could look at the first segment of the path, etc. etc. and do different things. I never quite thought about it that way.

So yeah, we could simply remove that History.started lock and allow multiple instances of LocationBar. I would just need to do a bit of investigation to see if there are any issues with [attaching multiple popstate / hashchange listeners](https://github.com/KidkArolis/location-bar/blob/2cab961fc5f1da9e240320cfac9f262d86876e64/location-bar.js#L169-L175) and such.

**Comment by raybooysen** [comment]
The only issue I see is my other issue which is #6 .  I.e. if you have many location bar instances, this would need to be solved, as each instance would expect it's state to be pushed after .start() is called.

**Comment by raybooysen** [comment]
Quoting @KidkArolis's response on #6 

```
But wouldn't you always execute start() only after first defining routes with route()?
In other words, you'll

create instance 1
route()
route()
start()

create instance 2
route()
route()
start()
start is local to the instance
```
If that is possible, then great.  The other thing I can think about is if you call ```locationbar.Route()``` will it overwrite any of the routes from other instances?

**Comment by KidkArolis** [comment]
Fixed via #7
